api/src/main/java/com/intern/api/client/Client.java

```diff
@@ -0,0 +1,30 @@
+package com.intern.api.client;
+
+import static com.mongodb.client.model.Filters.eq;
+
+import com.mongodb.client.MongoCollection;
+import java.util.ArrayList;
+import lombok.RequiredArgsConstructor;
+import org.bson.Document;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class Client {
```

Comment:
MongoClient

Reply:
This name already exist in mongo library. Renamed to DataBaseClient